ocs_ci/deployment/deployment.py

```diff
@@ -1263,10 +1263,16 @@ def deploy_lvmo(self):
         if "old" in file_version:
             file_version = file_version.split("-")[0]
 
-        cluster_config_file = os.path.join(
-            constants.TEMPLATE_DEPLOYMENT_DIR_LVMO,
-            f"lvm-cluster-{file_version}.yaml",
-        )
+        if "lvms" in config.DEPLOYMENT["ocs_registry_image"]:
+            cluster_config_file = os.path.join(
+                constants.TEMPLATE_DEPLOYMENT_DIR_LVMO,
+                "lvms-cluster.yaml",
```

Comment:
we should use same convention as `f"lvm-cluster-{file_version}.yaml"` imo

Reply:
@shyRozen @AviadP  any thoughts , should i rename the file?

Reply:
As the product is moving on, the new qe decide there convention. As we had a default to lvmo, this will be the default to lvms. I'm resolving the issue.